src/index.spec.ts

```diff
@@ -1,12 +1,176 @@
+import { rest } from 'msw';
+import { setupServer } from 'msw/node';
 import {
   isInteger,
   toLowerCase,
   removeDuplicatesFromArray,
   createRandomProduct,
   getStarWarsPlanets,
-  createProduct 
+  createProduct,
 } from './index';
 
-test('Testing time!', () => {
-  console.log('welcome');
+let noInput: any;
+const emptyString: string = '';
+const emptyArray: any = [];
+const emptyObject: any = {};
+const randomString: any = 'halsfhals';
+const randomNumber: any = 123131;
```

Comment:
Generally speaking. avoid the use of `any` as possible as you can. In these cases you could've skipped the any, specially for `randomString` and `randomNumber`.

Reply:
Got it. I used `any` in some cases because otherwise I wouldn't get the tested function to throw an error, as TS wouldn't let me pass a different type. As for `randomString` and `randomNumber`, totally, I missed that 😅